sql/core/src/main/scala/org/apache/spark/sql/catalog/interface.scala

```diff
@@ -25,6 +25,14 @@ import org.apache.spark.sql.catalyst.DefinedByConstructorParams
 // Note: all classes here are expected to be wrapped in Datasets and so must extend
 // DefinedByConstructorParams for the catalog to be able to create encoders for them.
 
+/**
+ * A database in Spark, as returned by the `listDatabases` method defined in [[Catalog]].
```

Comment:
There are multiple of them so these links won't work.

On Tuesday, May 31, 2016, Jacek Laskowski <notifications@github.com> wrote:

> In sql/core/src/main/scala/org/apache/spark/sql/catalog/interface.scala
> <https://github.com/apache/spark/pull/13406#discussion_r65140165>:
>
> > @@ -25,6 +25,14 @@ import org.apache.spark.sql.catalyst.DefinedByConstructorParams
> >  // Note: all classes here are expected to be wrapped in Datasets and so must extend
> >  // DefinedByConstructorParams for the catalog to be able to create encoders for them.
> >
> > +/**
> > + * A database in Spark, as returned by the `listDatabases` method defined in [[Catalog]].
>
> Use [[Catalog#listDatabases]]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/spark/pull/13406/files/798329a37b8690657e3210c5d0db17b6064c1b9c#r65140165>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AATvPL84lRsioxEvoYyBOZ1x1KqxScVvks5qG_BOgaJpZM4IqP2G>
> .
>